tools/fsd/.shed.yml

```
@@ -0,0 +1,6 @@
+categories: [Graphics]
+description: Tool that plots a histogram of sizes of read families
+long_description: Tool that plots a histogram of sizes of read families
+name: family_size_distribution
```

```suggestion
name: duplex_family_size_distribution
```
might be a better name ? Or `duplexanalysis_family_size_distribution` ?
`family_size_distribution ` seems a bit vague. Not a must, but I think it would help